Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+# Install required packages and remove the apt packages cache when done.
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        git \
+        python3 \
+        python3-dev \
+        python3-setuptools \
+        python3-pip \
+        nginx \
+        cron \
+        supervisor && \
+        pip3 install -U pip setuptools && \
```

Comment:
It should be moved 4 spaces to the left. But do we even need this?

Reply:
Upgrading `pip` and `setuptools` might be a good idea?

Reply:
I was thinking since the script installs pip and setuptools just above this, we probably don't need to upgrade or do anything.

Reply:
I just checked the build logs:
```
Installing collected packages: pip, setuptools
Found existing installation: pip 9.0.1
Not uninstalling pip at /usr/lib/python3/dist-packages, outside environment /usr
Found existing installation: setuptools 39.0.1
Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
Successfully installed pip-19.1.1 setuptools-41.0.1
```
Seems like this command is actually upgrading. Maybe `apt-get` does not install the latest version, so we upgrade using `pip3`?

Reply:
I just checked myself and it is indeed upgrading. You can ignore the comment now. Thank you,